spec.html

```diff
@@ -35129,6 +35129,7 @@ <h1>%TypedArray%.prototype.slice ( _start_, _end_ )</h1>
           1. If _relativeEnd_ &lt; 0, let _final_ be max((_len_ + _relativeEnd_), 0); else let _final_ be min(_relativeEnd_, _len_).
           1. Let _count_ be max(_final_ - _k_, 0).
           1. Let _A_ be ? TypedArraySpeciesCreate(_O_, &laquo; _count_ &raquo;).
+          1. If IsDetachedBuffer(_O_.[[ViewedArrayBuffer]]) is *true*, throw a *TypeError* exception.
```

Comment:
Previous behaviour was to throw iff `count > 0`. Similar to how for example [%TypedArray%.prototype.copyWithin](https://tc39.es/ecma262/#sec-%typedarray%.prototype.copywithin) works (cf. step 11.c). [%TypedArray%.prototype.fill](https://tc39.es/ecma262/#sec-%typedarray%.prototype.fill) was the only outlier which always throws even when no elements are touched (cf. step 10).

Reply:
Addressed.